src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,156 @@
+package kitchenpos.application;
+
+import static kitchenpos.domain.MenuFixture.*;
+import static kitchenpos.domain.MenuProductFixture.*;
+import static kitchenpos.domain.ProductFixture.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+
+@DisplayName("메뉴 서비스 테스트")
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+    @Mock
+    private MenuDao menuDao;
+    @Mock
+    private MenuGroupDao menuGroupDao;
+    @Mock
+    private MenuProductDao menuProductDao;
+    @Mock
+    private ProductDao productDao;
+
+    @InjectMocks
+    private MenuService menuService;
+
+    @DisplayName("메뉴 등록 API - 가격 없음")
+    @Test
+    void create_price_null() {
+        // given
+        MenuProduct menuProduct = menuProductParam(1L, 2L);
+        Menu menu = menuParam("후라이드+후라이드", null, 1L, Collections.singletonList(menuProduct));
+
+        // when, then
+        assertThatThrownBy(() -> menuService.create(menu))
+            .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("메뉴 등록 API - 가격 0원")
+    @Test
+    void create_price_zero() {
+        // given
+        MenuProduct menuProduct = menuProductParam(1L, 2L);
+        Menu menu = menuParam("후라이드+후라이드", BigDecimal.ZERO, 1L, Collections.singletonList(menuProduct));
+
+        // when, then
+        assertThatThrownBy(() -> menuService.create(menu))
+            .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("메뉴 등록 API - 메뉴 그룹 존재 X")
```

Comment:
```suggestion
    @DisplayName("메뉴 등록 API - 메뉴 그룹 존재 하지 않음")
```
의미전달에는 이슈가 없긴합니다만, 가능하면 기호 없이 표현해주셔도 좋을 것 같습니다..!